writing-specifications.rst

```diff
@@ -44,7 +44,7 @@ Example
 Specification Heading
 ^^^^^^^^^^^^^^^^^^^^^
 
-A Spec must begins with a spec heading and a single specification can
+A spec must begins with a spec heading and a single specification can
```

Comment:
Isn't `Spec` a noun here?

Reply:
Please disregard this. It is a common noun, not proper. Need my coffee.

Reply:
Ha ha. No worries. Thanks for merging